anykernel.sh

```diff
@@ -86,7 +86,7 @@ remove_section init.oem.debug.rc "service oemasserttip" "disabled"
 # Remove packet filtering from WCNSS_qcom_cfg.ini
 cp -pf /system/vendor/etc/wifi/WCNSS_qcom_cfg.ini $ramdisk/renderzenith/WCNSS_qcom_cfg.ini
 cp -pf /vendor/etc/wifi/WCNSS_qcom_cfg.ini $ramdisk/renderzenith/WCNSS_qcom_cfg.ini
-remove_line WCNSS_qcom_cfg.ini g_enable_packet_filter_bitmap
+remove_line $ramdisk/renderzenith/WCNSS_qcom_cfg.ini g_enable_packet_filter_bitmap
```

Comment:
Always glad to have another pair of eyes to look over the work we do. Sometimes things as small as this can slip through. This should fix delayed notifications.